addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -108,6 +108,54 @@
         .o_kanban_record {
             width: 350px;
         }
+        &.o_kanban_ungrouped {
+            padding: 0;
+            .o_mailing_list_kanban_grouped {
+                display: none!important;
+            }
+        }
+        &.o_kanban_grouped .o_mailing_list_kanban_ungrouped {
+            display: none!important;
+        }
+        .o_mailing_list_kanban_ungrouped {
+            .o_mailing_list_kanban_stats > div {
+                min-width: 70px;
+            }
+            .o_mailing_list_kanban_stats > a {
+                flex-wrap: wrap;
+                @include media-breakpoint-down(xs) {
+                    font-size: 1.4rem;
+                    height: 40px;
+                    margin-bottom: 30px;
+                    min-width: 90px;
+                    width: 30%;
+                }
+            }
+            .o_mailing_list_kanban_button,
+            .o_mailing_list_kanban_button:focus,
+            .o_mailing_list_kanban_button:hover {
```

Comment:
Any reason to specify focus and hover style here?
Especially since below it will become `.o_mailing_list_kanban_button:hover:hover` if I'm not mistaken?